stubs/docker/docker/utils/decorators.pyi

```diff
@@ -1,6 +1,6 @@
-from _typeshed import Incomplete
 from collections.abc import Callable
+from typing import Any
 
 def check_resource(resource_name: str): ...
 def minimum_version(version: str): ...
-def update_headers(f: Callable[..., Incomplete]): ...
+def update_headers(f: Callable[..., Any]): ...
```

Comment:
Considering that this is a decorator, we should either use  type var and use the same type var for the returned callable, or just leave it as `Incomplete` for now.